From a8aa1cda6cb7b7a500f34cafb61d7eba9c619d0c Mon Sep 17 00:00:00 2001
From: Jonathan Wakely <jwakely@redhat.com>
Date: Wed, 18 Jun 2025 15:46:24 +0100
Subject: [PATCH] libstdc++: Fix docs for --enable-vtable-verify [PR120698]

libstdc++-v3/ChangeLog:

	PR libstdc++/120698
	* doc/xml/manual/configure.xml: Do not claim that vtv is enabled
	by default.
	* doc/html/manual/configure.html: Regenerate.

(cherry picked from commit d199a9c7c5034d0eddb3380a58342a5bcbe6febd)
---
 libstdc++-v3/doc/html/manual/configure.html | 2 +-
 libstdc++-v3/doc/xml/manual/configure.xml   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/libstdc++-v3/doc/html/manual/configure.html b/libstdc++-v3/doc/html/manual/configure.html
index 3564b0c409f0..eb4d42117100 100644
--- a/libstdc++-v3/doc/html/manual/configure.html
+++ b/libstdc++-v3/doc/html/manual/configure.html
@@ -268,7 +268,7 @@
         operations (e.g. the library is configured for armv7 and then code
         is compiled with <code class="option">-march=armv5t</code>) then the program
         might rely on support in libgcc to provide the atomics.
-    </p></dd><dt><span class="term"><code class="code">--enable-vtable-verify</code>[default]</span></dt><dd><p>Use <code class="code">-fvtable-verify=std</code> to compile the C++
+    </p></dd><dt><span class="term"><code class="code">--enable-vtable-verify</code></span></dt><dd><p>Use <code class="code">-fvtable-verify=std</code> to compile the C++
     runtime with instrumentation for vtable verification. All virtual
     functions in the standard library will be verified at runtime.
     Types impacted include <code class="classname">locale</code> and
diff --git a/libstdc++-v3/doc/xml/manual/configure.xml b/libstdc++-v3/doc/xml/manual/configure.xml
index cd5f44458a2e..460b994009da 100644
--- a/libstdc++-v3/doc/xml/manual/configure.xml
+++ b/libstdc++-v3/doc/xml/manual/configure.xml
@@ -438,7 +438,7 @@
     </para>
  </listitem></varlistentry>
 
- <varlistentry><term><code>--enable-vtable-verify</code>[default]</term>
+ <varlistentry><term><code>--enable-vtable-verify</code></term>
  <listitem>
     <para>Use <code>-fvtable-verify=std</code> to compile the C++
     runtime with instrumentation for vtable verification. All virtual
-- 
2.43.7


